src/client.js

```diff
@@ -70,6 +70,8 @@ export class StreamChat {
 			delete this.options.httpsAgent;
 		}
 
+		this.request = axios.create(this.options);
```

Comment:
Can we rename this to something like `this.axiosInstance`, to not confuse it with javascript's request object?